modules/monitoring-applying-custom-alertmanager-configuration.adoc

```diff
@@ -57,7 +57,7 @@ The default value is 30 seconds (`30s`).
 <2> The `group_interval` value specifies how much time must elapse before Alertmanager sends a notification about new alerts added to a group of alerts for which an initial notification was already sent.
 The default is five minutes (`5m`).
 <3> The `repeat_interval` value specifies the minimum amount of time that must pass before an alert notification is repeated.
-The default is four hours (`4h`).
+The default setting for the Cluster Monitoring Operator is 12 hours (`12h`).
```

Comment:
4h is the default value in Alertmanager. CMO today will provision a default Alertmanager configuration which happens to have a 12h value but I wouldn't mention it in the docs since it could change in the future and existing clusters wouldn't see the new default.

TL;DR: I'd remove the default values for all interval  parameters.

Reply:
I removed all default value information.